test/test_openapi.py

```diff
@@ -120,6 +121,48 @@ def route2():
                 pass
         self.assertEqual(str(cm.exception), 'Only specify path_spec once per route path')
 
+    def test_shared_path_spec(self):
+        """
+        Assert that, when sharing the path_spec, routes don't overwrite each
+        other's properties.
+        """
+        app = AzulChaliceApp('testing', spec={'foo': 'bar'})
+        swagger_test_path_spec = {
+            'parameters': [
+                params.query('foo', schema.optional({'type': 'string'})),
+            ]
+        }
+
+        @app.route('/swagger-test-1', methods=['GET'], cors=True, path_spec=swagger_test_path_spec, method_spec={
+            'summary': 'Swagger test 1',
+        })
+        def swagger_test_1():
+            pass
+
+        @app.route('/swagger-test-2', methods=['GET'], cors=True, path_spec=swagger_test_path_spec, method_spec={
```

Comment:
I'd wrap this line. The amount of duplicated code is the same if one line or multiple.

Reply:
> Would a loop work?

The loop works! I don't know why I didn't think to try this.

> I'd wrap this line.

This line is wrapped already. Do you want